sql/org_mozilla_firefox/event_types_v1/query.sql

```diff
@@ -0,0 +1,15 @@
+DECLARE submission_date DATE DEFAULT @submission_date;
```

Comment:
This had to be a script because view definitions cannot contain query parameters.